models/marts/enriched_history/enriched_history_operations.yml

```diff
@@ -426,6 +426,8 @@ models:
           - accepted_values:
               values: [true, false]
               quote: false
+              config:
+                where: closed_at > current_timestamp - interval 2 day
```

Comment:
nit: I'm pretty sure the `accepted_values` test can be removed. This field is a boolean (only possible values are true, false, null) and there is already a `not_null` check

Reply:
Where would boolean check be enforced though? Is there a schema validator?